src/main/java/frc/trigon/robot/constants/AutonomousConstants.java

```diff
@@ -1,15 +1,27 @@
 package frc.trigon.robot.constants;
 
+import com.pathplanner.lib.auto.NamedCommands;
 import com.pathplanner.lib.path.PathConstraints;
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.trigon.robot.commands.CommandConstants;
+import frc.trigon.robot.commands.factories.AutonomousCommands;
+import frc.trigon.robot.commands.factories.ShootingCommands;
 
 public class AutonomousConstants {
     public static final PathConstraints REAL_TIME_CONSTRAINTS = new PathConstraints(2.5, 2.5, 4, 4);
+    public static final Rotation2d EJECT_FROM_SHOOTER_PITCH = Rotation2d.fromDegrees(0);
+    public static final double EJECT_FROM_SHOOTER_SPEED = 0.5;
```

Comment:
The should be in `ShootingConstants` and should be named according to how the other ones are named there

Reply:
But it's only used in Autonomous

Reply:
It's still a shooting constant. Also, it could technically be used in game as well